### ISSUE TITLE
After some minutes the audio stream is interrupted. A re-connect helps for another few minutes.

### ISSUE THREAD

**Issue by pleasantbud** [corrados/jamulus#199]
I’m using the most current jamulus on a Windows 7 machine. M-Audio M-Track with native ASIO DRIVERS. 
After a few minutes I can’t hear anything coming back in the headphones. I disconnect from the server and reconnect and I’m good for another few minutes. Usually can hang in for one full song.

**Comment by corrados** [comment]
Is the audio signal suddenly interrupted or is it a slow process where you have a lot of audio drop outs and finally the signal is cut off?
These "few minutes": Could you use a stop watch if it is always exactly the same time?
Do you have special firewalls running?

**Comment by pleasantbud** [comment]
The audio signal just suddenly stops coming back. The other members hear me. It seems rather random. Sometimes I play Through an entire song and sometimes not.
I don’t use a firewall, or use Kaspersky
I generally turn off the virus protection

Bud

On May 15, 2020, at 3:52 PM, Volker Fischer <notifications@github.com> wrote:

﻿

Is the audio signal suddenly interrupted or is it a slow process where you have a lot of audio drop outs and finally the signal is cut off?
These "few minutes": Could you use a stop watch if it is always exactly the same time?
Do you have special firewalls running?

**Comment by corrados** [comment]
One more question: If you cannot hear any audio from the server, does your buffer LED turns to red and stays red all the time?

**Comment by corrados** [comment]
To be able to help you, I need my above question to be answered (since I cannot reproduce your issue).

**Comment by pleasantbud** [comment]
So sorry and thanks for your willingness to help.
My group has not done it recently So At this time I can’t give you an answer.

Bud

On Jun 1, 2020, at 4:54 PM, Volker Fischer <notifications@github.com> wrote:

﻿

To be able to help you, I need my above question to be answered (since I cannot reproduce your issue).

**Comment by seannmcevenue** [comment]
I have been experiencing the same pattern with my 2010 iMac, 3.06 ghz, i3, 12 Gb ram, High Sierra. when I run Jamulus.  When I connect with my public server, it runs well for a minute or so and then the audio abruptly quits. The level indicators continue to operate. The buffer light goes to red and stays that way. There are random bursts of sound that stop as quickly as they started. The sound does not return unless I disconnect and reconnect, and repeat.

 I can bootup my Windows laptop and set up a Jamulus public server. Jamulus runs well that way. If I then connect with that server but from Jamulus on my iMac, it runs well indefinitely.

Thoughts? Thanks!

Sean

**Comment by corrados** [comment]
@seannmcevenue Sorry for the late response. When I read your problem description I immediately thought that your internet connection has a problem. Have you tried to use a lower audio quality and use mono? Do you still observe the problem with these settings?

**Comment by pleasantbud** [comment]
I haven’t tried in in quite a while.
My circumstances changed.

Buddy Coughlin
www.youtube.com/pleasantbud
www.pleasantbud.com
Sent from my too expensive device

On Aug 13, 2020, at 5:03 PM, Volker Fischer <notifications@github.com> wrote:

﻿

@seannmcevenue<https://github.com/seannmcevenue> Sorry for the late response. When I read your problem description I immediately thought that your internet connection has a problem. Have you tried to use a lower audio quality and use mono? Do you still observe the problem with these settings?

**Comment by seannmcevenue** [comment]
Thank you for your reply. Actually I had made those settings changes before my email but without any change to the problem. Everything worked fine when I was on someone else’s public server but not on mine.  The problem did get resolved but I am not sure how. My best guess is that after I deleted the server and app software and then  downloaded and set them up again, the problem resolved. I did this a number of times before the problem actually and finally resolved.
Thanks again, Jamulus has been a life saver!
Sean McEvenue
